pallets/subtensor/src/rpc_info/neuron_info.rs

```diff
@@ -147,6 +147,23 @@ impl<T: Config> Pallet<T> {
         Some(neuron)
     }
 
+    pub fn get_neuron_certificate(netuid: u16, uid: u16) -> Option<NeuronCertificate> {
```

Comment:
if we dont have rpcs, do we still need this ?

Reply:
cleaned up